Common/Logger.cs

```diff
@@ -23,16 +23,16 @@ public Logger(IFileWriteHandler FileWriteHandler, IEnvironmentVariables environm
             logFileName = $"{DateTime.Now.Day}.{DateTime.Now.Month}.{DateTime.Now.Year}.{DateTime.Now.Hour}.{DateTime.Now.Minute}.{DateTime.Now.Second}";
         }
 
-        location = $"bin/pirate{version}/logs";
         version = environmentVariables.GetVariable("version");
```

Comment:
What happens if there is no version in the config....maybe make a default version?